src/borg/testsuite/repository.py

```diff
@@ -856,7 +856,7 @@ class RemoteRepositoryTestCase(RepositoryTestCase):
     repository = None  # type: RemoteRepository
 
     def open(self, create=False):
-        return RemoteRepository(Location('__testsuite__:' + os.path.join(self.tmppath, 'repository')),
+        return RemoteRepository(Location('__testsuite__:' + os.path.join(self.tmppath, 'repository')+"::archive"),
```

Comment:
why is this needed?

Reply:
Sorry, this was left over from something I was testing. Ill be more careful about this.